go.mod

```diff
@@ -2,4 +2,24 @@ module github.com/Amenokal-Labs/mev-mantis.git
 
 go 1.18
 
-require github.com/joho/godotenv v1.5.1 // indirect
+require github.com/ethereum/go-ethereum v1.12.0
+
+require (
```

Comment:
I think depencendies from this line can be removed. you only need `go-ethereum` as dependency. others are dependencies of your dependency

Reply:
Compiler is not happy when removing those dependencies:
`go: updates to go.mod needed; to update it:
        go mod tidy`
When running the command above, it writes the dependencies i deleted back